src/editor.hpp

```diff
@@ -29,8 +28,9 @@ class Editor {
 	void deleteCharL();
     void deleteCharR();
 	
-	void addSimpleStatus(const std::string& name, const std::string& status_message);
-	void enableSimpleStatus(const std::string& name);
+	void printColoredStatus(const std::string& message, const int& colorPair);
```

Comment:
`const int&`? just do `int`

Reply:
breaks colors, don't change it

Reply:
Ah yeah, you can, fixed this in 3a712f55a6ee9142680da600eaaef86a4389eccf